simulation_parameters/Constants.cs

```diff
@@ -740,6 +740,11 @@ public static class Constants
     // Cooldown for AI for toggling engulfing
     public const float AI_ENGULF_INTERVAL = 300;
 
+    /// <summary>
+    ///   Probability, rolled at each AI step (which very often), that the AI will try to engulf something it can't eat
```

Comment:
I'm still seeing the "(which very often)" part which doesn't sound grammatically correct to me here.

Reply:
And now the line is too long :laughing: